s3/install

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+
+ARCH="$(uname -m)"
+OS="$(uname)"
+
+# convert to standard arch names used in files
+if [ "x${ARCH}" = "xx86_64" ]; then
+    ARCH="amd64"
+fi
+
+# convert to standard os names used in files
+# TODO: support Windows
```

Comment:
The bash script is probably hard to run on Windows and should be removed.

Reply:
You mean we don't need to care about Windows through this quickstart installer?